eva-lib/src/test/java/uk/ac/ebi/eva/lib/configuration/MongoRepositoryTestConfiguration.java

```diff
@@ -19,63 +19,42 @@
 package uk.ac.ebi.eva.lib.configuration;
 
 import com.github.fakemongo.Fongo;
-import com.mongodb.Mongo;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.mongodb.MongoClient;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.convert.converter.Converter;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.data.mongodb.MongoDbFactory;
-import org.springframework.data.mongodb.config.AbstractMongoConfiguration;
-import org.springframework.data.mongodb.core.convert.CustomConversions;
-import org.springframework.data.mongodb.core.convert.DbRefResolver;
-import org.springframework.data.mongodb.core.convert.DefaultDbRefResolver;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.SimpleMongoDbFactory;
 import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
-import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
 import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
 
-import uk.ac.ebi.eva.commons.models.converters.data.DBObjectToVariantEntityConverter;
-
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
+import uk.ac.ebi.eva.lib.MongoConfiguration;
 
 @Configuration
 @EnableMongoRepositories(basePackages = "uk.ac.ebi.eva.lib.repository")
-public class MongoRepositoryTestConfiguration extends AbstractMongoConfiguration {
-
-    @Autowired
-    private MongoDbFactory mongoDbFactory;
-
-    @Override
-    protected String getDatabaseName() {
-        return "test-db";
-    }
+@Import({MongoConfiguration.class})
+@PropertySource({"classpath:eva.properties"})
+public class MongoRepositoryTestConfiguration {
```

Comment:
The previous version of this class was mixing too many things, and overlapped with MongoConfiguration. Once we removed redundancy across other classes, the `AbstractMongoConfiguration` was just a constraint on what methods to override, and was not really needed.